app/workers/expire_rent_email_worker.rb

```diff
@@ -0,0 +1,8 @@
+class ExpireRentEmailWorker
+  include Sidekiq::Worker
+
+  def perform
+    rents = Rent.where(end_date: Time.zone.today)
+    rents.map { |rent| RentMailer.expire_rent_send(rent.id).deliver_now }
```

Comment:
```suggestion
    rents.each { |rent| RentMailer.expire_rent_send(rent.id).deliver_now }
```

Could be an `each` because you are not transforming the array, but it's just a  minor issue

Reply:
Done